lib/yabeda/sidekiq.rb

```diff
@@ -34,6 +34,10 @@ module Sidekiq
       gauge     :active_processes,     tags: [],        comment: "The number of active Sidekiq worker processes."
       gauge     :queue_latency,        tags: %i[queue], comment: "The queue latency, the difference in seconds since the oldest job in the queue was enqueued"
 
+      gauge     :concurrency,          tags: [],        comment: "The total number of jobs that can be run at a time across all processes."
+      gauge     :available_workers,    tags: [],        comment: "The number of workers available for new jobs across all processes."
```

Comment:
From the metric name I can't understand whether it is about processes or threads. I need to read the collector's code below to get it. Let's clarify:
```suggestion
      gauge     :available_threads,    tags: [],        comment: "The number of threads available for job processing across all processes."
```

Reply:
This name was chosen to be consistent with how it’s reporting active workers — it looks like Sidekiq uses the term “worker” to mean “thread”.